examples/globe_wfs_extruded.js

```diff
@@ -40,6 +40,20 @@ function setMaterialLineWidth(result) {
     }
 }
 
+function altitudeLine(properties, contour) {
+    var altitudes = [];
+    var i = 0;
```

Comment:
Why put it here and not in the `for` loop ?

Reply:
Because of the lint.
[eslint] All 'var' declarations must be at the top of the function scope. (vars-on-top)

Reply:
Ah, yeah, the lint, I forgot about it. Thanks for the information.